### PR TITLE
Adding condor support for sysbatcher

### PR DESCRIPTION
**Pull request by gmadigan**
Nearly identical to its counterpart but obviously submits jobs with condor rather than batch.

### CODE REVIEW

**Comment by DavidMorse** [comment]
SysBatcher.py already runs on HTCondor, so I'm not sure why a new file is needed here. As far as I can tell only the year and brag arguments were added, which could easily have been added to SysBatcher.py. Am I missing something?

**Comment by gmadigan** [comment]
Weird. You're totally right, I must have gotten confused (these are changes I made ages ago that I only now have sorted into a dedicated branch). Did it always use condor? Anyway, I moved the changes to SysBatcher.py and removed the other script, which is reflected in a new commit to this branch.

**Comment by DavidMorse** [comment]
It runs on Condor since you updated it last year :)

https://github.com/DavidMorse/LQ2Analysis/commit/d94ebb0c767aa1b023a460bbdea8a9524b994278

File: SysBatcher.py [hunk 1]
```diff
@@ -20,7 +37,7 @@
 
 
 # Loop over channels and systematic variations
-for c in ['uujj','uvjj']:
+for c in ['uujj']:
```
Comment (DavidMorse) [inline SysBatcher.py]: please just put back the original line commented out so we remember how to add it back later

**Review by DavidMorse — CHANGES_REQUESTED** [review]

File: SysBatcher.py (hunk 1 above)
Comment (gmadigan) [reply SysBatcher.py]: Ok, I fixed it. See if that looks good.

**Pull request merged**